api_session.yaml

```
@@ -134,6 +166,10 @@ definitions:
     $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v21.5.3/openapi/definitions.yaml#/FiscalCode"
   ProblemJson:
     $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v21.5.3/openapi/definitions.yaml#/ProblemJson"
+  UnlockCode:
+    type: string
+    minLength: 9
+    maxLength: 9
```

We should use a regex pattern instead.
```suggestion
  UnlockCode:
    type: string
    pattern: ^\d{9}$
```